src/main/webapp/i18n/en/quizExercise.json

```diff
@@ -226,7 +226,17 @@
                 "title": "Short Quiz Title",
                 "questionTitle": "Question Title",
                 "shortQuestionTitle": "Short Question Title"
-            }
+            },
+            "matchLetterCase": {
+                "title": "Match Letter Case",
+                "explanation": "Select if the letter case of the answers provided matters in the evaluation (Example: Answer /= answer)"
+            },
+            "matchPercentage": {
+                "title": "Match Answers Exactly",
+                "explanationTitle": "Similarity %",
+                "explanation": "Determines the similarity percentage value, for which the answers are treated as correct (Default 100%)"
```

Comment:
```suggestion
                "explanation": "Determines the required similarity between predefined solutions and student solutions in the evaluation for each spot. The value (between 50% and 100%) indicates whether only exactly correct answers (100%) are treated as correct or whether students can have typo(s) in their solutions. The lower the value is, the more typos are allowed in the student solutions (based on the Levenshtein distance https://en.wikipedia.org/wiki/Levenshtein_distance). Example: a value of 85% would allow 1 typo in a word with 5 characters and 2 typos in a word with 10 characters. Please use the preview feature for the quiz if you would like to try out different values."
```